store/store-core/src/main/java/org/eclipse/lyo/store/internals/SparqlStoreImpl.java

```diff
@@ -502,97 +514,106 @@ private Model modelFromQueryFlatPaged(final URI namedGraph, final URI type, fina
     }
 
     //This method currently only provides support for terms of type Comparisons, where the operator is 'EQUALS', and the operand is either a String or a URI.
-    private SelectBuilder constructSparqlWhere(final String prefixes, final String where, final String searchTerms, final int limit, final int offset) {
+    private SelectBuilder constructSparqlWhere(final String prefixes, final String where, final String searchTerms, final int limit, final int offset, List<String> additionalDistinctVars, SelectBuilder additionalQueryFilter) {
 
-    	SelectBuilder distinctResourcesQuery = new SelectBuilder();
+        SelectBuilder distinctResourcesQuery = new SelectBuilder();
 
-    	//Setup prefixes
-    	Map<String, String> prefixesMap = new HashMap<String, String>();
+        //Setup prefixes
+        Map<String, String> prefixesMap = new HashMap<String, String>();
         try {
-        	if (!StringUtils.isEmpty(prefixes)) {
-    			prefixesMap = QueryUtils.parsePrefixes(prefixes);
-    	        for (Entry<String, String> prefix : prefixesMap.entrySet()) {
-    	            distinctResourcesQuery.addPrefix(prefix.getKey(), prefix.getValue());
-    			}
-        	}
-		} catch (ParseException e) {
+            if (!StringUtils.isEmpty(prefixes)) {
+                prefixesMap = QueryUtils.parsePrefixes(prefixes);
+                for (Entry<String, String> prefix : prefixesMap.entrySet()) {
+                    distinctResourcesQuery.addPrefix(prefix.getKey(), prefix.getValue());
+                }
+            }
+        } catch (ParseException e) {
             throw new IllegalArgumentException("prefixesExpression could not be parsed", e);
-		}
+        }
 
         distinctResourcesQuery
         .addVar( "s" )
-		.setDistinct(true)
-		.addWhere( "?s", "?p", "?o");
+        .setDistinct(true)
+        .addWhere( "?s", "?p", "?o");
 
+        if (null != additionalDistinctVars) {
+            for (String var : additionalDistinctVars) {
```

Comment:
var is a reserved keyword since java 10 https://developers.redhat.com/blog/2018/05/25/simplify-local-variable-type-definition-using-the-java-10-var-keyword

Reply:
This I believe is fixed now.